flink-core/src/main/java/org/apache/flink/configuration/ClusterOptions.java

```diff
@@ -132,11 +132,7 @@
                             "Defines whether the cluster uses fine-grained resource management.");
 
     public static boolean isDeclarativeResourceManagementEnabled(Configuration configuration) {
```

Comment:
I thought about going one step further and removing this method and the `ENABLE_DECLARATIVE_RESOURCE_MANAGEMENT` option, but `ENABLE_FINE_GRAINED_RESOURCE_MANAGEMENT` makes this quite annoying so I left it as is.

Reply:
I don't fully understand why it is not possible. I thought that the declarative resource management is now always enabled. Hence `isFineGrainedResourceManagementEnabled` could be changed by removing the first if check which checks if declarative resource management is not activated. Then we should be able to remove the `ENABLE_DECLARATIVE_RESOURCE_MANAGEMENT` config option.